testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/hibernate/sessionfactorytest/SessionFactoryTestCase.java

```diff
@@ -53,6 +55,12 @@ public class SessionFactoryTestCase {
 
     private static final String ARCHIVE_NAME = "jpa_sessionfactory";
 
+    // This test needs to be recompiled against Hibernater ORM 6 (WFLY-16178) in order to pass.
+    @BeforeClass
+    public static void beforeClass() {
+        assumeTrue(System.getProperty("ts.ee9") == null);
```

Comment:
You need to check for ts.bootable.ee9 as well as that enables the profile used in jobs like https://ci.wildfly.org/viewType.html?buildTypeId=WF_MainNightlyJobs_WildFlyPreviewBootableJarLinuxJdk11

Reply:
Thanks Brian, added!